Ginger/GingerCore/Drivers/SeleniumDriver.cs

```diff
@@ -314,23 +331,32 @@ public override void StartDriver()
                         }
                         else
                         {
-                            if (Convert.ToInt32(HttpServerTimeOut) > 60)
+                            if (Convert.ToInt32(HttpServerTimeOut) >= 60)
                             {
                                 InternetExplorerDriverService service = InternetExplorerDriverService.CreateDefaultService();
+
+                                if (HideConsoleWindow == true)
+                                    service.HideCommandPromptWindow = true;
+
```

Comment:
few issues here:
1- there is no handling in case the IE type is 64bit- see row 323 If block
2- because you set the HttpServerTimeOut condition to be >= it means that you modified the defualt handling because till now it was considered only if it was bigger than the defualt 60
3- in case user set the timeout to let's say 30 and set HideConsoleWindow to "true" then current code won't do HideConsoleWindow because main If will be false

you should modify the If to be: if (Convert.ToInt32(HttpServerTimeOut) != 60  || HideConsoleWindow == true)
and inside If block to handle either one of them or both of them as required